model/articles.model.js

```diff
@@ -0,0 +1,33 @@
+const db = require("../db/connection");
+const data = require("../db/data/test-data/index");
+const format = require("pg-format");
+
+
+function selectArticlesById(articleId) {
+    return db.query(`SELECT * FROM articles WHERE article_Id = $1`, [articleId])
+      .then((result) => {
+        if (result.rows.length === 0) {
+          return Promise.reject({status: 404, msg: "article does not exist"})
+        }
+        return result.rows[0];
+      });
+  }
+  
+  function selectArticles() {
+    return db.query(`SELECT articles.article_id, articles.title, articles.author, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.body) 
```

Comment:
comment count is currently being returned as a string which means we'd have to turn it into a number on the frontend which isn't ideal. To solve this, we can turn the comment count into an integer in the count line!

We can use the cast operator (::) in sql to convert a value to another type
We use it like this : `value::target_type`
try using this to convert the COUNT(comments.body) into an INTEGER